lib/perfdata/influxdbwriter.cpp

```diff
@@ -487,7 +499,7 @@ void InfluxdbWriter::Flush()
 	url->SetPath(path);
 
 	url->AddQueryElement("db", GetDatabase());
-	url->AddQueryElement("precision", "s");
+	url->AddQueryElement("precision", "ns");
```

Comment:
What happens on the InfluxDB side when you change the precision for metrics from `s` to `ns` for existing data?

Reply:
For existing data this does not change anything, only new points will be inserted with the higher precision.

InfluxDB uses `ns` as the default percision via its REST API and the CLI interface. So we think it makes sense to store the the performance data with this precision and not articifially reduce the precision of the stored data. Of course, it would also make sense to have a configuration option for this.

Reply:
We have fixed this on its own in #9599, please rebase.

Reply:
This precision consumes an unacceptable amount of disk space.